recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/google/snappy.git
+  git_tag: {{ version }}
+
+requirements:
+  build:
+    - msinttypes  # [win]
+    - libtool     # [unix]
+    - automake    # [unix]
+    - pkg-config  # [unix]
+
+about:
+  home: https://github.com/google/snappy
+  summary: A fast compressor/decompressor
+  license: Other
```

Comment:
Looks like BSD 3-Clause. Could we put that here? I know the data is licensed differently, but don't think we want to distribute that anyways.